WalletWasabi/WabiSabi/Models/MultipartyTransaction/ConstructionState.cs

```diff
@@ -13,6 +13,8 @@ public ConstructionState(MultipartyTransactionParameters parameters)
 		{
 		}
 
+		public TimeSpan ConnectionConfirmationTimeout => Parameters.ConnectionConfirmationTimeout;
```

Comment:
I think this belongs in the round state, it hasn't got much to do with the on chain structure which is what the multiparty transaction model is meant to abstract.

This is basically only for the side effect of serialization if i understand correctly, I'm fine with it for now but it shouldn't stay here for long.